props/saber_BC_buttons.h

```diff
@@ -775,15 +794,14 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
       return true;
 
 // Turn Saber ON Muted
-    case EVENTID(BUTTON_POWER, EVENT_FOURTH_HELD, MODE_OFF):
-      if (!mode_volume_) {
-        if (SetMute(true)) {
-          unmute_on_deactivation_ = true;
-          scroll_presets_ = false;
-          On();
-        }
+case EVENTID(BUTTON_POWER, EVENT_FOURTH_HELD, MODE_OFF):
+    if (!mode_volume_) {
+      if (SFX_mute) {
```

Comment:
This will do nothing if there is no mute sound, I don't think that's what you want.

Reply:
ah I didn't test with no mute sound.
ok I'll be back.

Reply:
ok fixed.